### PR TITLE
Set-up project structure and deploying to heroku

### PR DESCRIPTION
**Pull request by uche-inyama**
Hi TSE, :wave: 

In this milestone, we have:

 - Set up the basic structure of the file for implementing our project.
 - We have also deployed to heroku; use the link on README to check it out, it just blank screen for now.

Best, 

Team.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "name": "redux-bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.1"
```
Comment (bellom) [inline package.json]: image
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896806/
You missed those 👆 
Kindly add it and resubmit for re-review.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 
### I hope you're not working alone on this project, if so, kindly explain the reason why.

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: package.json (hunk 1 above)
Comment (moudio) [reply package.json]: We are not working alone, I was having some issues with my create-react-app. Thanks

**Review by AdnanAfsari — APPROVED** [review]
**Status: Approved**

Hello @MouhaDiouf  @uche-inyama ✋ 

Looks good now so it's a 👍 from me.

Happy Coding
Adnan Afsari

**Pull request merged**